src/py42/services/util.py

```diff
@@ -13,3 +15,18 @@ def get_all_pages(func, key, *args, **kwargs):
         yield response
         page_items = response[key]
         item_count = len(page_items)
+
+
+def escape_quote_chars(token):
```

Comment:
In general, this might make more sense at the root `util.py`.
However, I like that it is here because this way, it won't show up in the docs! and it can be treated as an implementation detail and changes at will for v1 of py42.